website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -104,54 +104,57 @@ models:
-- `apply` (default) &mdash; attempts to update the existing database object if possible, avoiding a complete rebuild. 
-- `continue` &mdash; allows runs to continue while also providing a warning that the model was not executed
-- `fail` &mdash; forces runs to fail if a change is detected in a materialized view
+  - **v1.6.0:** Introduced support for `materialized_view`
 
-Additionally, you can apply the `auto_refresh` configuration to have Redshift [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you. This action can applied without the need to rebuild the materialized view. 
+</Changelog>
 
-You can create a materialized view by editing _one_ of these files:
-- the SQL file for your model
-- the `dbt_project.yml` configuration file
+The Redshift adapter supports [materialized views](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html).
+Redshift-specific configuration includes the typical `dist`, `sort_type`, `sort`, and `backup`.
+For materialized views, there is also the `auto_refresh` setting, which allows Redshift to [automatically refresh](https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-refresh.html) the materialized view for you.
+The remaining configuration follows the general [materialized view](/docs/build/materializations#Materialized-View) configuration.
+There are also some limitations that we hope to address in the next version.
 
-The following examples create a materialized view: 
+### Monitored Configuration Changes
 
-<File name='models/YOUR_MODEL_NAME.sql'>
+The settings below are monitored for changes applicable to `on_configuration_change`.
 
-```sql
-{{
-  config(
-    materialized = 'materialized_view',
-    auto_refresh = False,
-    on_configuration_change = 'apply',
-  )
-}}
-```
+#### Dist
 
-</File>
+Changes to `dist` will result in a full refresh. Redshift requires a materialized view to be
+dropped and recreated to apply a change to the `distkey` or `diststyle`.
 
+#### Sort Type, Sort
 
-<File name='dbt_project.yml'>
+Changes to `sort_type` or `sort` will result in a full refresh. Redshift requires a materialized
+view to be dropped and recreated to apply a change to the `sortkey` or `sortstyle`.
 
-```yaml 
-models:
-  path:
-    materialized: materialized_view
-```
-</File>
+#### Backup
+
+Changes to `backup` will result in a full refresh. Redshift requires a materialized
+view to be dropped and recreated to apply a change to the `backup` setting.
+
+#### Auto Refresh
+
+The `auto_refresh` setting can be updated via an `ALTER` statement. This setting effectively toggles
+automatic refreshes on or off. The default setting for this config is off (`False`). If this
+is the only configuration change for the materialized view, dbt will choose to apply
+an `ALTER` statement instead of issuing a full refresh,
```

Comment:
If `on_configuration_change = "continue"` and there no changes, yes, this should issue a manual refresh. This happens because we only get to the `on_configuration_change` setting if there are changes (which there are not in this scenario).

With that in mind, if there ***are*** configuration changes, and `on_configuration_change = "continue"`, should we also refresh there? We are not currently doing that; we currently do nothing and move on.

Reply:
I think we should in the future yes

Reply:
This is captured in a separate story and the related docs updates are part of that story. This won't be resolved here.